presto-tablestore/src/main/java/com/facebook/presto/tablestore/TablestoreConfig.java

```diff
@@ -0,0 +1,116 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.tablestore;
+
+import com.facebook.airlift.configuration.Config;
+
+public class TablestoreConfig
+{
+    private String endpoint;
+    private String accessKeyId;
+    private String accessKeySecret;
+    private String instance;
+    private int clientRetrySeconds = 120;
+    private int connectTimeoutSeconds = 10;
+    private int socketTimeoutSeconds = 50;
+    private int syncClientWaitTimeoutSeconds = 60;
```

Comment:
Are these settings configurable by users? If yes, could you add documentation of these settings in the alibaba cloud tablestore doc?

Reply:
yes, they are configurable, but I'd prefer merge this first PR into prestodb first, and later I can expose more settings for user to tune.